x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_integrations.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { memo, useMemo } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiAccordion,
+  EuiTitle,
+  EuiPanel,
+  EuiButtonIcon,
+  EuiBasicTable,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { Agent, AgentPolicy, PackagePolicy, PackagePolicyInput } from '../../../../../types';
+import { useLink } from '../../../../../hooks';
+import { PackageIcon } from '../../../../../components';
+import { displayInputType, getLogsQueryByInputType } from './input_type_utils';
+
+const StyledCollapsablePanel = styled(EuiPanel).attrs((props) => ({
+  paddingSize: 'none',
+}))`
+  .euiAccordion__triggerWrapper {
```

Comment:
I do not think it's possible to do it without custom css here, I know @hbharding implemented it the same way for the overview panel,

Reply:
@elastic/eui-design Do you think there is anything built in that I can use to build this component? I currently use an EuiAccordion in a EuiPanel with some custom css

<img width="679" alt="Screen Shot 2020-11-30 at 1 26 08 PM" src="https://user-images.githubusercontent.com/1336873/100650584-2817ac00-3312-11eb-8b93-5a0be74545fa.png">
<img width="656" alt="Screen Shot 2020-11-30 at 1 26 03 PM" src="https://user-images.githubusercontent.com/1336873/100650586-2817ac00-3312-11eb-9b5b-2a227483ff3f.png">

Reply:
Hey @nchaulet , It looks like what you're actually building is a table within the contents of the accordion body. I would explicitly use the EuiBasicTable component so that all the right associations are made between cells. Then you'll just need to remove the padding from the accordion but pass a custom padding and class to the button portion of the accordion. Here's an example CodeSandox https://codesandbox.io/s/thirsty-river-hmfd2?file=/index.js

Reply:
Thanks a lot I just updated to use a buttonClass in EuiAccordion works perfectly :)